streaming/text/convert/enwiki/mds/create_pretraining_data.py

```diff
@@ -19,7 +19,8 @@
 flags.DEFINE_string("input_file", None,
                     "Input raw text file (or comma-separated list of files).")
 
-flags.DEFINE_string("output_dir", None, "Output example directory.")
+flags.DEFINE_string("local", None, "Local directory path to store the output MDS shard files")
```

Comment:
Inputs are prefixed `input_`, we need to call these `output_`

Also this breaks `make_train_parallel.py`

Reply:
Nice catch! Addressed as part of https://github.com/mosaicml/streaming/pull/171/commits/5b5c10450637d4ccdbc8b5c2403a7d18f89b169b.